go/obscuronode/enclave/enclave.go

```diff
@@ -36,6 +36,7 @@ type enclaveImpl struct {
 	exitCh               chan bool
 	speculativeWorkInCh  chan bool
 	speculativeWorkOutCh chan speculativeWork
+	genesisRollup        *Rollup
```

Comment:
umm, not sure if this is the place to store the rollup. It should be in the database. 
Basically, the first time the node starts, it will find the genesis, and store it in the db, ( and then store the other rollups on top of it), but there should be a pointer to the genesis.